src/components/PeopleTable.tsx

```diff
@@ -1,5 +1,35 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-export const PeopleTable = () => {
+import { useContext } from 'react';
+import { PeopleContext } from '../store/PeopleContent';
+import { PersonLink } from './PersonLink';
+import { SearchLink } from './SearchLink';
+import classNames from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+
+export const PeopleTable: React.FC = () => {
+  const { peopleToShow } = useContext(PeopleContext);
+  const [searchParams] = useSearchParams();
+
+  const sortParam = searchParams?.get('sort');
+  const orderParam = searchParams?.get('order');
+
+  const sortOption = (sortOptionParam: string) => {
+    if (!sortParam || (sortParam && !orderParam)) {
+      return sortOptionParam;
+    } else if (sortParam && orderParam && sortParam !== sortOptionParam) {
+      return sortOptionParam;
+    }
+
+    return null;
+  };
+
+  const orderOption = (sortOptionParam: string) => {
+    if (sortParam && !orderParam && sortOptionParam === sortParam) {
+      return 'desc';
+    }
+
+    return null;
```

Comment:
The `orderOption` function returns `null` by default, which might not be the intended behavior. Consider returning a default order, such as 'asc', when no specific order is set.